test/metabase/automagic_dashboards/core_test.clj

```diff
@@ -113,6 +112,46 @@
       (with-dashboard-cleanup
         (-> card-id Card (automagic-analysis {}) valid-dashboard?)))))
 
+(expect
```

Comment:
Would be good to know what this stuff is testing. Is the fact that it's a `valid-dashboard?` sufficient?